backend/index.ts

```diff
@@ -1,17 +1,19 @@
-import dotenv from "dotenv";
-dotenv.config();
 import { corsMiddleware, logger } from "./middlewares/middlewares";
 import express, { Express } from "express";
 
+import SwaggerUI from "swagger-ui-express";
 import activities from "./routes/activities";
 import connectToMongoDB from "./config/mongoClient";
 import { createServer } from "https";
 import { createWebsocketServer } from "./config/websocket";
-
+import dotenv from "dotenv";
 import { readFileSync } from "fs";
 import surveys from "./routes/surveys";
+import { swaggerJs } from "./config/SwaggerOptions";
 import users from "./routes/users";
 
+dotenv.config();
```

Comment:
Załadowanie dotenv'u powinno być na początku pliku, ponieważ inne importy również korzystają z ładowanych zmiennych. Teraz m.in. connectToMongo nie będzie znał danych do połączenia z mongo